tests/JustSaying.UnitTests/Messaging/Channels/Fakes/FakeMessageDeleter.cs

```diff
@@ -0,0 +1,14 @@
+using System.Threading;
+using System.Threading.Tasks;
+using JustSaying.Messaging.Channels.Context;
+
+namespace JustSaying.UnitTests.Messaging.Channels.Fakes
+{
+    public class FakeMessageDeleter : IMessageDeleter
```

Comment:
I admittedly tend towards fakes instead of mocks in general, as fakes only need to be created once. In this case we don't really care about tracking actions (we track them on the ISqsQueue instead), so it's simple enough that a mock would require no setup true. Happy to change it, I don't mind too much 🙂